src/worker.py

```
@@ -1697,6 +1697,301 @@ def _persist_attachment(self, session_id, info):
+                "hand_raised": existing["hand_raised"],
+            }),
+            exclude_session_id=session_id,
+        )
+
+        try:
+            return Response(None, status=101, web_socket=client)
+        except TypeError:
+            # CPython tests use a minimal Response stub without web_socket support.
+            return Response(None, status=101)
+
+    async def on_webSocketMessage(self, ws, message):
```

Ruff (0.15.10)

[warning] Too many return statements (7 > 6)

(PLR0911)